### ISSUE TITLE
Version bump for v7.11.2

### ISSUE THREAD

**Issue by toanmxpixta** [o19s/elasticsearch-learning-to-rank#365]
Generally available ES versions of ElasticCloud deployments are 7.11.2 and 7.12.
It's time to upgrade Elasticsearch Learning to Rank to newer version.

Release notes
https://www.elastic.co/guide/en/elasticsearch/reference/7.11/release-notes-7.11.2.html

**Comment by GretelDePaepe** [comment]
Is the plugin going to be available for es 7.12 soon?  http://es-learn-to-rank.labs.o19s.com/

**Comment by nathancday** [comment]
Hello @toanmxpixta and @GretelDePaepe,

As I'm sure you know Elasticsearch follows a very fast release cadence. And we do our best to keep the plugin in close to the current version, but it's a lot of work. We are testing a new development cycle for the plugin, where we will release a version for the X.X.1 and ask the community to pitch in with the patch versions after that.

Would either of you consider contributing a PR for es7.11.2? I'd be happy to go over the process with you to bring you into the project as contributors.

Cheers,
Nate

**Comment by worleydl** [comment]
@toanmxpixta 7.11.2 didn't require any code changes so I went ahead and pushed that release.

@GretelDePaepe 7.12.0 support was released yesterday however our releases are now being hosted on github.  I will update the docs to reflect that.  This is atypical as Nate mentioned but we just happened to have the code ready for this one.

https://github.com/o19s/elasticsearch-learning-to-rank/releases/tag/v1.5.4-es7.11.2
https://github.com/o19s/elasticsearch-learning-to-rank/releases/tag/v1.5.4-es7.12.0
